services/src/main/java/org/keycloak/services/resources/admin/RealmAdminResource.java

```diff
@@ -773,4 +775,28 @@ public void clearUserCache() {
         }
     }
 
+    /**
+     * Extension point where an SPI can provide a custom implementation for a certain admin REST path.
+     *
+     * @param path the (unmapped) path name
+     * @return the rest resource or null if no implementation found
+     */
+    @Path("{unmapped_path}")
+    public Object resolveUnknowPath(@PathParam("unmapped_path") String path) {
```

Comment:
Please remove adding custom REST endpoints. It's beyond the scope of this work and we have someone else working on it already.